File: src/main/java/br/com/southsystem/assembleia/repositorio/VotacaoRepository.java
package br.com.southsystem.assembleia.repositorio;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import br.com.southsystem.assembleia.entidade.Votacao;

@Repository
public interface VotacaoRepository extends JpaRepository<Votacao, Long> {

	Optional<Votacao> findByVotoIdPautaId(Long idPauta);

	Optional<Votacao> findByVotoIdPautaIdAndVotoIdAssociadoId(Long idPauta, Long idAssociado);

}

File: src/main/kotlin/br/com/southsystem/desafiobackvotos/gateway/database/VotacaoRepository.kt
package br.com.southsystem.desafiobackvotos.gateway.database

import br.com.southsystem.desafiobackvotos.domain.Votacao
import org.springframework.data.repository.reactive.ReactiveCrudRepository

interface VotacaoRepository: ReactiveCrudRepository<Votacao, Long> {
}


